### PR TITLE
data capture

### PR DESCRIPTION
**Pull request by Rodrigmi**
Have added the minimum number files necessary to run the data capture
on the fetch robot by adding this to your catkin workspace

### CODE REVIEW

**Review by julianweisbord — CHANGES_REQUESTED** [review]
I think that all of these files should probably just go under the path: Deep-Learning-For-Object-Recognition-on-a-Mobile-Robot/src/image_capture/

Also remember to add instructions to the readme dependencies. Start with a line that says to install ROS indigo then mention the catkin workspace, changes to the .bashrc, etc.

**Comment by julianweisbord** [comment]
Any update to this?

**Comment by Rodrigmi** [comment]
No I haven't worked on it since Thursday because I went home for the weekend to spend time with my parents for my moms birthday.

**Comment by Rodrigmi** [comment]
Just updated the dependencies in the root directory and moved the lifelong_object_learning package into image_capture.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,30 @@ For our senior design capstone project, we will build an image classifier on top
...
+
+Create catkin_ws by running the following commands:
+
+	mkdir -p ~/catkin_ws/src
+	cd ~/catkin_ws/
+	catkin_make
+
+Add /src/image_capture/lifelong_object_learning package to the robots catkin_ws/src directory
+
+Add following two lines to your .bash_rc
```
Comment (julianweisbord) [inline README.md]: Add the following

**Comment by julianweisbord** [comment]
Since your updating the location of data_collect.py, delete the original file at: Deep-Learning-For-Object-Recognition-on-a-Mobile-Robot/src/image_capture/data_collect.py
Can you delete the imgs folder at the root directory?
Other than that everything looks good!

**Comment by julianweisbord** [comment]
Also add a "results" folder at the base of our repo and within that put a "setup" folder. We should place our setup mapping files there instead of in "Deep-Learning-For-Object-Recognition-on-a-Mobile-Robot/src/image_capture/lifelong_object_learning/mapping/"
Also remember to add your sequence of commands from one note to the readme :)

**Comment by Rodrigmi** [comment]
Made the changes Julian requested, should be all good now.

**Comment by julianweisbord** [comment]
.gitignore files depend on the operating system so its uncommon to push a gitignore which may need to be different from person to person, but other than that this looks great!

**Review by julianweisbord — DISMISSED** [review]

**Comment by Rodrigmi** [comment]
Ok I didn’t know that, should I remove it then before we merge it?

**Comment by julianweisbord** [comment]
That'd be great!

**Comment by Rodrigmi** [comment]
Removed the .gitignore file

**Review by mccallm1 — APPROVED** [review]
Initial changes to code looks nice and clean, and reworking of file structure aids the logical flow of the pipeline.

**Pull request merged**